scripts/agent-evals/src/runner/paths.ts

```diff
@@ -0,0 +1,11 @@
+import path from "path";
+import { fileURLToPath } from "url";
+
+export function getAgentEvalsRoot(): string {
+  const thisFilePath = path.dirname(fileURLToPath(import.meta.url));
+  return path.resolve(path.join(thisFilePath, "..", ".."));
```

Comment:
I don't love that these path.resolves() depend so tightly on exactly where files are in the directory structure (in a way that is more obscure than normal imports), but I can't think of a better approach, so 🤷

Reply:
I know... I realized there were a few place that needed these paths, so I put it all in one file in anticipation that this will break. Haven't thought of a better plan yet